purview/utils/general.py

```diff
@@ -178,7 +178,7 @@ def create_query(item, uid_name, uid, reverse=False):
     return query
 
 
-def query_neo4j(query):
+def query_neo4j(query, resource):
```

Comment:
This function was written to be pretty generic and by adding this parameter it makes it specific to what the story API needs. How about making this `resources_to_expand` and have it be an optional list of labels the developer can provide that will have `serialized_all` be called on?

Reply:
hm, how about I make an optional param which will have to be a list, and if it's present then '''python query_neo4j ''' will expand that node, else no.

Reply:
Oh, I think we are telling the same thing. Sorry!